go.mod

```diff
@@ -14,7 +15,7 @@ require (
 	k8s.io/apimachinery v0.19.7
 	k8s.io/cli-runtime v0.19.7
 	k8s.io/client-go v0.19.7
-	k8s.io/code-generator v0.19.7
+	k8s.io/code-generator v0.20.1
```

Comment:
@rhuss I'm not quiet sure why `go mod tidy` bumps generator version as well. I've pinned it to match the rest of k8s dependencies.

Reply:
yeah, that is always tricky to find out. I suspect that there is a direct reference to `code-generator` somewhere and the go dep resolution always picks the highest number.

Reply:
However, since this is a build time dependency I don't expect any issue if the code compiles and test passes.

Reply:
Yep, we have the reference in hack/tools.go to help with Export type generation etc.

Reply:
So it's `buildpacks -> different version of go-containerregistry@v0.4.0 -> newer code-generator`. However, as you mentioned it's build time dependency and we don't rely on any code generation refresh here, should we keep the newer one?

Reply:
Per our chat keeping the knative/pkg version of code-generator https://github.com/openshift/knative-client/pull/694/commits/97666596f1186e8cf73b53f7908f6aad98dc4e56.